tests/internal/anvil/framework.go

```diff
@@ -46,7 +46,7 @@ const (
 	failStr = "--- FAIL:"
 
 	// Standard timeout used in SDK calls in tests
-	StdTimeOut = 5 * time.Second
+	StdTimeOut = 10 * time.Second
```

Comment:
RSA keys are quite slow to create and/or use. I was thinking of putting out a little PR to move Anvil to using a single set of keys for a test run to speed up things a little.